src/assets/scripts/math/flightMath.js

```diff
@@ -155,3 +160,14 @@ export const dist_to_boundary = (airport, pos) => {
     // TODO: hmm, `position.position`? that seems fishy
     return abs(distance2d(pos, airport.position.position) - airport.ctr_radius);
 };
+
+/**
+ *
+ *
+ *
+ * @function calculateHeadingFromTwoPositions
+ * @param positionEnd {array}
+ * @param positionStart {array}
+ * @return {number}
+ */
+export const calculateHeadingFromTwoPositions = (positionEnd, positionStart) => vradial(vsub(positionEnd, positionStart));
```

Comment:
You're right, however, I merely renamed a previous implementation so it would be inline with current usages.  So the order of properties came from that.  

I agree, though, that the order is confusing.  I'll add a comment here to clarify the why and hopefully that help make things a little clearer.